api/DocumentOrShadowRoot.json

```diff
@@ -275,7 +277,7 @@
               "version_added": "41"
             },
             "safari": {
-              "version_added": true
+              "version_added": "12"
             },
             "safari_ios": {
               "version_added": true
```

Comment:
```suggestion
              "version_added": "12"
```

Reply:
This removes some whitespace on the left, so I'll make my own commit.

Reply:
Not a problem!  Actually the whitespace thing seems to be an odd bug in GitHub -- it's not actually removing a space, but GitHub thinks it is.  (I've run into it on another PR, and when adding a space to satisfy GitHub, it turns out it was adding an extra space.)